packages/reactivity/src/reactive.ts

```diff
@@ -106,9 +113,19 @@ export declare const ShallowReactiveMarker: unique symbol
 export type ShallowReactive<T> = T & { [ShallowReactiveMarker]?: true }
 
 /**
- * Return a shallowly-reactive copy of the original object, where only the root
- * level properties are reactive. It also does not auto-unwrap refs (even at the
- * root level).
+ * Creates a shallowly-reactive copy of the original object.
```

Comment:
Like `reactive()` this should be changed (https://vuejs.org/api/reactivity-advanced.html#shallowreactive)

Reply:
still remains